### PR TITLE
chore(color) - Fix minor layout issues arising from libopenui PR 89.

### PR DESCRIPTION
**Pull request by philmoz**
EdgeTX/libopenui#89 makes the number edit and text edit box height consistent with other UI elements.

This introduces some minor layout issues that are fixed in this PR.

This should be merged in conjunction with PR 89 from libopenui.

### CODE REVIEW

File: radio/src/gui/colorlcd/preview_window.h [hunk 1]
```diff
@@ -181,21 +181,18 @@ class ThemedTextEdit : public TextEdit
...
     }
 
     void paint(BitmapBuffer *dc) override
     {
       colorMaintainer.applyColorValues();
+      lv_obj_set_style_bg_color(lvobj, makeLvColor(COLOR(_colorBackgroundIndex)), LV_PART_MAIN);
+      lv_obj_set_style_text_color(lvobj, lv_color_white(), LV_PART_MAIN);
       FormField::paint(dc);
-
-      dc->drawSizedText(FIELD_PADDING_LEFT, FIELD_PADDING_TOP, value, length, COLOR(_colorTextIndex));
```
Comment (gagarinlg) [inline radio/src/gui/colorlcd/preview_window.h:198]: The text is not displayede anymore?

**Review by gagarinlg — COMMENTED** [review]
Same here is in libopenui: we need to centralise the styling

File: radio/src/gui/colorlcd/preview_window.h (hunk 1 above)
Comment (philmoz) [reply radio/src/gui/colorlcd/preview_window.h:198]: Value is displayed by the TextEdit control - this line was trying to overwrite the value in a different color.
The 'Edit' and 'Focus' boxes did not reflect how they actually look - now they should.

Comment (gagarinlg) [reply radio/src/gui/colorlcd/preview_window.h:198]: ok, strange

**Review by gagarinlg — APPROVED** [review]

**Pull request merged**